scripts/reds-deploy.sh

```diff
@@ -53,12 +53,23 @@ main() {
     done
 
     # Verify the commands we are going to use are present ahead of time
+    assert_cmd_exists aws
     assert_cmd_exists mktemp
     assert_cmd_exists mkdir
     assert_cmd_exists rm
     assert_cmd_exists tar
     assert_cmd_exists unzip
 
+    # Check if the user has the proper version of aws cli installed
+    local aws_version_output
+    aws_version_output=$(aws --version);
+    if [[ ! "$aws_version_output" =~ aws-cli/1.* ]]; then
```

Comment:
[nit,optional] Quotes are not necessary for variables within `[[ ... ]]`.
```suggestion
    if [[ ! $aws_version_output =~ aws-cli/1.* ]]; then
```

Reply:
Hope you are right 
5d092d3